### DISCUSSION TITLE
Please help: First time set up lighthouse, drone drifting

### DISCUSSION THREAD

**Discussion by Zm442709994** [bitcraze/discussions#651] [Q&A] [answered]
First time step up the system with two used HTC Vive 1.0 base stations. The drone’s accelerometer reading goes crazy when I put it inside the lighthouse area, the coordinate drift as well. Anybody knows why? Much appreciated!
https://youtu.be/U0Knsvgfxvs

**Comment by krichardsson** [comment]
When it looks crazy in the client like this, it is probably because the lighthouse system gets conflicting data for some reason. When "random"/conflicting data is pushed into the kalman filter it tries to adapt and you get this behavior. The question is why the data is not good.

The first thing to check is if the base stations are synchronized. You can synchronize them either by cable or light, but you must set the modes accordingly. See [instructions from Vive](https://www.vive.com/us/support/vive-pro2/category_howto/installing-the-base-stations.html)

The second thing is the base station geometry. I think it looks OK in the video but it is hard to see. Check in the lighthouse tab if the base station positions seems to match where the base stations are located in the real world. Maybe you have to redo the geometry? 
If the geometry is not correct, the system will get different positions from the two base stations.

A third thing to check for (if the two above are OK) is reflections. I don't see any obvious reflective surfaces in your video, but if there is a mirror for instance, this can cause problems.

One experiment could be to turn off the second base station, I would expect the system to stabilize as it only gets information from one base station. This would indicate that one of the above is the problem.

**Reply by Zm442709994** [reply, marked as answer]
Thank you for your response.  The problem has been resolved.
